Applications/ConsumerApi/test/ConsumerApi.Tests.Integration/StepDefinitions/TokensApiStepDefinitions.cs

```diff
@@ -27,40 +27,33 @@ internal class TokensApiStepDefinitions
     private ApiResponse<EmptyResponse>? _createTokenResponse401;
     private ApiResponse<Token>? _tokenResponse;
     private ApiResponse<ListTokensResponse>? _tokensResponse;
-    private readonly HttpClient _httpClient;
-    private readonly ClientCredentials _clientCredentials;
     private bool _isAuthenticated;
-    private Client _sdk = null!;
+    private Client? _sdk;
```

Comment:
We want to call it `client` instead of `sdk`.

Reply:
changes made